omnibus/.kitchen.yml

```diff
@@ -57,6 +57,16 @@ platforms:
     run_list: apt::default
   - name: ubuntu-14.04
     run_list: apt::default
+  - name: ubuntu-14.10
```

Comment:
Out of curiosity why are the .10 releases included? Historically we've only built/tested on the LTS releases.

Reply:
It tripped me up a bit that there weren't *any* 16 or 18 platforms in the kitchen file.  I looked back at the pattern that already existed for 12.x and 13.x and made the assumption that the kitchen file should have all releases beyond just LTS.

Would you like me to go back and remove all of them except 16.04 and 18.04?

Reply:
Removing everything except 16.04 and 18.04 seems cleanest. Thanks!

Reply:
Fair enough, I've updated my branch to remove Ubuntu 10 - 14 leaving only the current LTS versions and kicked off the following ad-hoc build for good measure:

https://buildkite.com/chef/chef-opscode-pushy-client-master-omnibus-adhoc/builds/22